src/status_im/browser/core.cljs

```diff
@@ -189,12 +189,13 @@
   [cofx browser url]
   (let [history-index (:history-index browser)
         history       (:history browser)]
-    (let [new-history (conj (subvec history 0 (inc history-index)) url)
-          new-index   (dec (count new-history))]
-      (update-browser cofx
-                      (assoc browser
-                             :history new-history
-                             :history-index new-index)))))
+    (when history
+      (let [new-history (conj (subvec history 0 (inc history-index)) url)
+            new-index   (dec (count new-history))]
+        (update-browser cofx
+                        (assoc browser
+                               :history new-history
+                               :history-index new-index))))))
```

Comment:
This can be condensed using `when-let`

```clojure
(when-let [history (:history browser)]
  (let [history-index ...
        new-history ...
        new-index ...]
    ...))
```